algorithms/ar-kmp/python3/knuth_morris_pratt.py

```diff
@@ -0,0 +1,58 @@
+# Python program for KMP Algorithm 
+
+def LPSArray(pat, M, lps): 
+	len = 0 
+
+	lps[0]
```

Comment:
Codacy found an issue: [Statement seems to have no effect](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6363416)